src/plugins/dash/videojs-dash.js

```diff
@@ -298,7 +299,7 @@ Html5DashJS.hooks_ = {};
 
 const canHandleKeySystems = function(source) {
   // copy the source
-  source = JSON.parse(JSON.stringify(source));
+  source = assign({}, source);
```

Comment:
Are you sure that changing from deep-copy to shallow-copy is enough? it will keep nested properties by reference

Reply:
it fixes the IE issue and I didn't see an issue with other browsers after the change the problem is that there are circular references JSON.stringify in IE can not handle.